setup.py

```diff
@@ -1,38 +1,96 @@
+from __future__ import print_function
+
 import os
+import sys
+
+from Cython.Build import cythonize
+from setuptools import Command, Extension, setup
+
+
+COMPILER_DIRECTIVES = {}
+DEFINE_MACROS = None
+FLAG_COVERAGE = '--cython-coverage'  # custom flag enabling Cython line tracing
+BASEDIR = os.path.abspath(os.path.dirname(__file__))
+NAME = 'cftime'
+CFTIME_DIR = os.path.join(BASEDIR, NAME)
+CYTHON_FNAME = os.path.join(CFTIME_DIR, '_{}.pyx'.format(NAME))
+
+
+class CleanCython(Command):
+    description = 'Purge artifacts built by Cython'
+    user_options = []
 
-from setuptools import Extension, setup
+    def initialize_options(self):
+        pass
 
-rootpath = os.path.abspath(os.path.dirname(__file__))
+    def finalize_options(self):
+        pass
 
+    def run(self):
+        for rpath, _, fnames in os.walk(CFTIME_DIR):
+            for fname in fnames:
+                _, ext = os.path.splitext(fname)
+                if ext in ('.pyc', '.pyo', '.c', '.so'):
+                    artifact = os.path.join(rpath, fname)
+                    if os.path.exists(artifact):
+                        print('clean: removing file {!r}'.format(artifact))
+                        os.remove(artifact)
+                    else:
+                        print('clean: skipping file {!r}'.format(artifact))
```

Comment:
This is just a developer convenience. I got tired of manually cleaning out the Cython binary files, so now all the developer requires to do is::

```
$ python setup.py clean_cython
$ python setup.py build_ext --inplace
```